src/sentry/static/sentry/app/views/stream.jsx

```diff
@@ -623,17 +638,30 @@ const Stream = React.createClass({
   renderAwaitingEvents() {
     let org = this.getOrganization();
     let project = this.getProject();
+    let platformName = project.platform
+      ? getPlatformName(project.platform)
+      : 'JavaScript';
     let sampleLink = null;
     if (this.state.groupIds.length > 0) {
       let sampleIssueId = this.state.groupIds[0];
+
       sampleLink = (
         <p>
           <Link to={`/${org.slug}/${project.slug}/issues/${sampleIssueId}/?sample`}>
-            {t('Or see a sample Javascript event')}
+            {tct('Or see your sample %s event', platformName)}
```

Comment:
I'm not sure what you mean here?

Also, I'm thinking of removing `platformName` entirely, as in the event it's a Rails project, we don't actually have a Rails sample event—only a Ruby sample event. Passing this information from the server `/samples/*.json` to the frontend is complicated and not of huge value.